content/utmt/rooms/creating new rooms/_index.md

```diff
@@ -8,15 +8,39 @@ resources:
   - name: optimalRoomSide
     src: "optimalRoomSide.png"
     title: Optimal placing for left and right transitions
+  - name: RoomSetup
+    src: "RoomSetup.png"
+    title: This is how it should look now
+  - name: RoomOnCreate
+    src: "RoomOnCreate.png"
+    title: This is how it will look when first adding a new room
+  - name: ViewsSetup
+    src: "ViewsSetup.png"
+    title: If you've done everything correctly this is how it should look 
 ---
 
 TODO: having some more screenshots in here would be nice. Also fix some phrasing.
 
 ### Preparation:
-When creating a new room, the dimensions __must__ be multiples of `320` x `240`. Having different room dimensions will cause lots of troubles. Also make sure that the room's room speed is set to `60` as that's what AM2R runs at. Entering different values will lead to the room being slowed-down or sped up.  
-Once you have your set room size, set a background by putting `bgScreenGuide` in one of the background definitions by dragging and dropping it from the left. Enable the background afterwards.  
+Before we set up tiles or textures, we must first make some preperations, to get started, create a new room by right clicking on `Rooms` in the file viewer on the left and then click add. Now you will see 3 things; 
+A list of all the ojects that are in your room
+A set of values, corresponding to the object in your room that you currently have selected
+A Visual overview of your room, if you're adding a new room, this will be completely black with a grey grid.
+
+{{< img name="RoomOnCreate" size=origin >}}
+
+When creating a new room, make __absolutely__ sure the dimensions are multiples of `320` x `240`. Having different room dimensions will cause lots of troubles. Also make sure that the room's room speed is set to `60` as that's the Framerate what AM2R runs at. Entering different values will lead to the room being slowed-down or sped up.  
+Once you have your set room size, set a background by searching for `bgScreenGuide` and putting it in one of the background definitions by dragging and dropping it from the left. Enable the background afterwards.  
+You probably also want to give your room a name, for this, it is recommended to follow the naming convention of all the other rooms in the game, meaning that you start with `rm_` then the area your room belongs to (`a0`, `a1`, `a2`, etc..), a letter to denote which subsection of that area you are in (`h` for outside, and then `a`, `b`, etc...) and finally the number of the room itself (`01`, `02`, `03`, etc...) 
+e. g. If you were adding a Room inside Golden Temple, you would name it something like `rm_a1a13` 
```

Comment:
```suggestion
If you were adding a room inside Golden Temple for example, you would name it something like `rm_a1a13`. But of course you can use any names you like.  
```

Reply:
Don't forget to use two spaces at the end of a line to indicate a newline in markdown.